LCM/dsc/engine/ConfigurationManager/MSFT_DSCLocalConfigurationManager.c

```diff
@@ -34,7 +39,10 @@ void MI_CALL MSFT_DSCLocalConfigurationManager_Load(
     MI_UNREFERENCED_PARAMETER(selfModule);
 
     *self = NULL;
-
+#if defined(BUILD_OMS)
+    memset(&oldAction, 0, sizeof(struct sigaction));
```

Comment:
Would it be useful to additional messages while doing the signal handler specific operations? If you think that will be helpful while investigating, we should add those.

Reply:
may be we should add one on signal handler itself so we know if our handler is even called or not.

Reply:
incase of python we don't know if handler is called as that do not have logging.